packages/xstate-wallet/src/workflows/create-and-fund.ts

```diff
@@ -158,7 +169,11 @@ type Service = keyof ReturnType<typeof services>;
 
 const options = (store: Store) => ({
   services: services(store),
-  actions: {triggerObjective: triggerObjective(store), assignJointChannelId}
+  actions: {
+    triggerObjective: triggerObjective(store),
+    assignJointChannelId,
+    reserveFunds: reserveFunds(store)
```

Comment:
It seems important to know whether `reserveFunds` was successful or not. So, I think this should be an invoked service, rather than a (fire-and-forget) action.

Reply:
Good point